backend/functions/orchestration/fpl_scrapper.py

```
@@ -29,35 +30,106 @@ def run(self) -> None:
+            The current game week as a string, or None if it cannot be determined.
+        """
+        # Try and collect current game week
         self.logger.info('Determining current Game Week...')
-        gw = GameWeek().get_current()
-        self.logger.info(f'Current Game Week: {gw} \n')
+        try:
+            gw = GameWeek().get_current()
+
+            if not isinstance(gw, str) and "being updated" not in gw:
+                self.logger.info(f'Current Game Week: {gw} \n')
+            return gw
```

Ruff (0.14.2)

Consider moving this statement to an `else` block

(TRY300)